projects/cm_mcu/LocalTasks.c

```diff
@@ -1321,8 +1317,8 @@ int init_load_clk(int clk_n)
   }
 
   if (PreambleList_row == 0xff) {
-    log_warn(LOG_SERVICE, "Quit.. garbage EEPROM PreL\r\n");
-    return status_r; // fail reading and exit
+    log_warn(LOG_SERVICE, "Quit.. garbage EEPROM of %s PreL\r\n", clk_ids[clk_n]);
+    return status_r + 1; // fail reading and exit
```

Comment:
if I am reading the code right, `status_r=0` here. So maybe it's clearer just to return `1`?

Reply:
done.